### PR TITLE
feature/52

### PR DESCRIPTION
**Pull request by gkgbunny**
https://github.com/nodexpertsdev/RnD/issues/52#issue-433194349

### CODE REVIEW

**Comment by aman-successive** [comment]
It is good that you have put error and success messages in one file, but did you check whether these files are accessed properly where they are being used?
Kindly look into this.

**Comment by gkgbunny** [comment]
I will surely do that once it is approved.

**Comment by aman-successive** [comment]
Please do it in this issue, as the issue should not be only to create a file but to use it in the module

**Comment by aman-successive** [comment]
Kindly Remove conflicts

File: expressEngine/cms/infoMessage/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import infoMessage from './info';
+
+export { infoMessage };
```
Comment (chiragSuccessive) [inline expressEngine/cms/infoMessage/index.js]: single export file should be exported as default

File: expressEngine/order/service/order.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { BaseService, DBService } from '../../lib/service/index';
 import { Order } from '../../model/index';
```
Comment (chiragSuccessive) [inline expressEngine/order/service/order.js]: remove index from file name

File: expressEngine/product/service/product.js [hunk 3]
```diff
@@ -5,7 +5,11 @@ import { BaseService, DBService } from '../../lib/service';
 import { Product, ProductDetail } from '../../model';
 
 // import messages
+<<<<<<< HEAD
```
Comment (chiragSuccessive) [inline expressEngine/product/service/product.js]: remove this conflict comments

**Comment by mayank-nx** [comment]
@gkgbunny There are too many files and as per discussion, it is more than what you have worked on. I am closing this PR. Please open one with your changes only.

**Pull request closed without merging**